### DISCUSSION TITLE
[Chart] Tooltip should show total in Stacked charts (bar, area, ...)

### DISCUSSION THREAD

**Discussion by ValentinC-BR** [apache/superset#18462] [Ideas]
## Context

The current tooltip only show the value of the bar we're hovering on.
It is often (almost always ?) interesting to know the total value, and when there are many bars, displaying it at the top of the bars is not a viable option.

## Feature Request

Show total in tootltip :
* Propose it as an option in the UI would be even better
* Leave the possibility to the user to add % in the tooltip would also be great, but quite a different feature.

What do you think of it ?

**Comment by juanmantelli** [comment]
I agree. See total at the top of the bars (stacked) may be really useful

**Comment by kamalkeshavani-aiinside** [comment]
Legacy Area Charts had the total and percentage values in tooltip, but it is lost in echarts.

**Comment by j4m4l** [comment]
Total and percentage in case of stacked bar charts is very important feature and sometime a necessity. 
Any pointers on how we can start the implementation, we can give it a try.

**Comment by LeoDiep** [comment]
+ 1 on this, need it a lot

**Comment by 3mmdrew** [comment]
Yeah I agree, this would be incredibly helpful. If there any workarounds at the moment they would be much appreciated.

**Comment by ninaviereckel** [comment]
+1 on this. Came here when looking for instructions how to do this. Would really need this feature!

**Comment by clavidk** [comment]
+100 Yes pleaseeeee!

**Comment by Rusp0** [comment]
Totally agree, please!
I spent 2 days trying to understand how to visualize % and total value on Bar Chart, but unsuccesfull..

**Comment by Rusp0** [comment]
@ValentinC-BR @juanmantelli @kamalkeshavani-aiinside @j4m4l @LeoDiep @3mmdrew @ninaviereckel @clavidk
I asked to add this feauture 2 times in official Slack space, some guy created a request. 
Vote for that request https://github.com/apache/superset/discussions/28216

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/pull/27950
